x-pack/plugins/secops/server/graphql/kpi_network/resolvers.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SourceResolvers } from '../../graphql/types';
+import { AppResolverOf, ChildResolverOf } from '../../lib/framework';
+import { KpiNetwork } from '../../lib/kpi_network';
+import { createOptions } from '../../utils/build_query/create_options';
+import { QuerySourceResolver } from '../sources/resolvers';
+
+export type QueryKipNetWorkResolver = ChildResolverOf<
```

Comment:
Should this be `QueryKpiNetworkResolver`?

Reply:
👍